### PR TITLE
password reset fix

### PR DESCRIPTION
**Pull request by bashiribndawud**
password reset fix

### CODE REVIEW

File: src/configs/endpoints.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const authEndpoints = {
   signupCompany: `${baseEndpoints.auth}organizations/signup`,
   signinCompany: `${baseEndpoints.auth}organizations/login`,
   activate: `${baseEndpoints.auth}/register/activate`,
-  resetPassword: `${baseEndpoints.auth}password`,
```
Comment (acexode) [inline src/configs/endpoints.js:19]: this resetpassword endpoint is already being used elsewhere, before you change anything, first confirm if it is used elsewhere

File: src/sections/share-data/view-request.jsx [hunk 2]
```diff
@@ -36,20 +36,32 @@ const formatDate = (dateString) => {
...
     }
-    return obj.shareable_type; 
   });
 }
 
 
 const ViewRequest = ({ data, handleCloseModal }) => {
  
   const [loading, setLoading] = useState()
-  const types = mapShareableTypes(data?.shareable_informations)
```
Comment (acexode) [inline src/sections/share-data/view-request.jsx:52]: why is this being changed

**Review by acexode — CHANGES_REQUESTED** [review]

File: src/sections/share-data/view-request.jsx (hunk 2 above)
Comment (bashiribndawud) [reply src/sections/share-data/view-request.jsx:52]: shareable_informations is empty. the requested data is now in requested_info array

File: src/configs/endpoints.js (hunk 1 above)
Comment (bashiribndawud) [reply src/configs/endpoints.js:19]: Okay

**Review by acexode — APPROVED** [review]

**Pull request merged**